common/source/docs/common-gps-ardusimple.rst

```diff
@@ -0,0 +1,48 @@
+.. _common-gps-ardusimple:
+
+==========
+ArduSimple
+==========
+
+ArduSimple provide a gamme of GNSS receivers based on Ublox F9P module. Their modules allow to have RTK configuration, RTK Heading, and RTK Correction Service (NTRIP Caster).
+
+Using RTK allows ArduPilot to get its position with centimeter-level accuracy, which is much more accurate than can be provided by a normal GPS.
+
+Where to Buy
+============
+
+- `ArduSimple website <https://www.ardusimple.com/>`_
+
+
+Pin Map
+=======
+
+The JST-GH connector is following the Pixhawk standard:
+
+   -  1: 5V_IN
+   -  2: ZED-F9P UART1 RX (3.3V level)
+   -  3: ZED-F9P UART1 TX (3.3V level)
+   -  4,5: Not connected
+   -  6: GND
+
+Please note that the board only includes GPS and doesn’t include magnetometer.
+
+.. image:: ../../../images/ardusimple_connector.png
+    :target: ../_images/ardusimple_connector.png
+    :width: 450px
+
+Wiring and Connections
+======================
+
+All ArduSimple GNSS models come with a JST GH 6-pin connector/cable that is compatible with the Pixhawk family and many other autopilots.
```

Comment:
I think this "Wiring and Connections" section should be combined with the "Pin Map" section.